lua/starfall/libs_sh/darkrp.lua

```diff
@@ -0,0 +1,1111 @@
+		local requestsForPlayer = self.requests[sender]
+		if not requestsForPlayer then return end
+
+		local request = requestsForPlayer[receiver]
+		requestsForPlayer[receiver] = nil
+
+		if next(requestsForPlayer) == nil then
+			self.requests[sender] = nil
+			if next(self.requests) == nil then hook.Remove("Think","SF_DarkRpMoneyRequests") end
+		end
+		
+		if not request then return end
+		if request.instance.error then
+			printDebug("SF: Attempted to accept money request but the instance was dead.", request)
+			return
+		end
+
+		return request
+	end
+
+	-- Console commands for managing money requests
+	local function chatPrint(ply, ...)
+		local message = string.format(...)
+		if ply:IsValid() and ply:IsPlayer() then
+			ply:PrintMessage(HUD_PRINTCONSOLE, message)
+		else
+			print(message)
+		end
+	end
+
+	concommand.Add("sf_moneyrequest", function(executor, command, args)
+		local sender, action, receiver = tonumber(args[1]), args[2], tonumber(args[3])
+		if not sender or not action or not receiver then
+			return chatPrint(executor, "sf_moneyrequest: malformed parameters (do \"help sf_moneyrequest\")")
+		end
+		sender = sender == 0 and executor or Entity(sender)
+		if not (sender:IsValid() and sender:IsPlayer()) then
+			return chatPrint(executor, "sf_moneyrequest: invalid sender")
+		end
+		receiver = Entity(receiver)
+		if not (receiver:IsValid() and receiver:IsPlayer()) then
+			return chatPrint(executor, "sf_moneyrequest: invalid receiver")
+		end
+		if executor:IsValid() and sender ~= executor and not executor:IsSuperAdmin() then
+			return chatPrint(executor, "sf_moneyrequest: only superadmins can interact with other people's money requests")
+		end
+		local request = MoneyRequest:popRequest(sender, receiver)
+		if request then
+			if action == "accept" then
+				request:accept()
+			elseif action == "decline" then
+				request:decline()
+			elseif action == "info" then
+				chatPrint(executor, request:info())
+			else
+				chatPrint(executor, "sf_moneyrequest: invalid action")
+			end
+		else
+			chatPrint(executor, "sf_moneyrequest: no such request")
+		end
+	end, nil, "Accept, decline, or view info about a StarfallEx money request. Usage: sf_moneyrequest <entindex or 0> <accept|decline|info> <request index>", FCVAR_CLIENTCMD_CAN_EXECUTE)
+	concommand.Add("sf_moneyrequest_update", function(executor)
+		if executor:IsValid() and not executor:IsSuperAdmin() then
+			return
+		end
+		requestsUpdate()
+		chatPrint(executor, "sf_moneyrequest_update: done")
+	end, nil, "Manually trigger a purge of expired/invalid money requests. Superadmin/RCON only.", FCVAR_UNREGISTERED)
+	concommand.Add("sf_moneyrequest_purge", function(executor)
+		if executor:IsValid() and not executor:IsSuperAdmin() then
+			return
+		end
+		for k in pairs(requests) do
+			requests[k] = nil
+		end
+		chatPrint(executor, "sf_moneyrequest_purge: done")
+	end, nil, "Manually trigger a purge of all money requests. Superadmin/RCON only.", FCVAR_UNREGISTERED)
```

Comment:
Probably should delete these. Also requestUpdate() function doesn't exist

Reply:
`requestsUpdate` did exist, but you removed it in e64e2f9.